charts/xp-management/templates/_helpers.tpl

```diff
@@ -53,8 +53,8 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 app.kubernetes.io/part-of: caraml
-{{- if .Values.extraLabels -}}
-    {{ toYaml .Values.extraLabels -}}
+{{- if .Values.extraLabels }}
+{{ toYaml .Values.extraLabels }}
```

Comment:
Ah lol oops I totally see where I had a typo in the copy paste ><" Thanks for identifying this!

Reply:
npnp